site/src/examples/carousel/Autoplay.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+  Button,
+  FlexLayout,
+  Link,
+  StackLayout,
+  Text,
+  useId,
+} from "@salt-ds/core";
+import {
+  Carousel,
+  CarouselAnnouncement,
+  CarouselAutoplayIndicator,
+  CarouselCard,
+  type CarouselEmblaApiType,
+  CarouselNextButton,
+  CarouselPreviousButton,
+  CarouselProgressLabel,
+  CarouselSlides,
+} from "@salt-ds/embla-carousel";
+import { PauseIcon, PlayIcon } from "@salt-ds/icons";
+import type { EmblaCarouselType } from "embla-carousel";
+import { default as AutoplayPlugin } from "embla-carousel-autoplay";
+import Classnames from "embla-carousel-class-names";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { sliderData } from "./exampleData";
+import styles from "./index.module.css";
+
+const DELAY_MSECS = 8000;
+const SETTLE_PIXEL_THRESHOLD = 10;
```

Comment:
Is this supposed to be used?

Reply:
SETTLE_PIXEL_THRESHOLD can be deleted

Reply:
deleted... was needed when we used settle and now use select